tests/Cluster/SettingsTest.php

```diff
@@ -38,7 +40,7 @@ public function testSetReadOnly(): void
             $index->addDocument($doc2);
             $this->fail('should throw read only exception');
         } catch (ClientResponseException $e) {
-            $error = \json_decode($e->getResponse()->getBody(), true)['error'] ?? null;
+            $error = \json_decode((string) $e->getResponse()->getBody(), true)['error']['root_cause'][0] ?? null;
```

Comment:
Do you know why this changed?

Reply:
that's a good catch:
Seems like elasticsearch response looks like:
```
{
  "error": {
    "type": "cluster_block_exception",
    "root_cause": [
      {
        "type": "cluster_block_exception",
        ...
      }
    ]
  }
}
```

Reply:
I wonder why the test passed before 🤔  If I remember correctly, there was a major ES version where the response format changed, but that was quite a while ago. Good to know we have it covered in the tests.